python/ray/autoscaler/v2/autoscaler.py

```diff
@@ -76,6 +79,7 @@ def __init__(
         self._metrics_reporter = metrics_reporter
 
         self._init_cloud_instance_provider(config, config_reader)
+        self.cloud_resource_monitor = None
```

Comment:
```suggestion
        self._cloud_resource_monitor = None
```

Align with the naming style.